assets/scss/6-components/table/table.html

```diff
@@ -1,30 +1,26 @@
-<table class="c-table {{ className }}">
+<table class="c-table {{ className }} t-sans l-width-full">
```

Comment:
Open sans would be inherited in most cases so I'm thinking we don't need it in our demo

Reply:
Gone.